client/data/sites/site-excerpt-constants.ts

```diff
@@ -10,8 +10,14 @@ export const SITE_EXCERPT_REQUEST_FIELDS = [
 	'name',
 	'options',
 	'plan',
+	'jetpack',
 ] as const;
 
 export const SITE_EXCERPT_COMPUTED_FIELDS = [ 'slug' ] as const;
 
-export const SITE_EXCERPT_REQUEST_OPTIONS = [ 'is_wpforteams_site', 'updated_at' ] as const;
+export const SITE_EXCERPT_REQUEST_OPTIONS = [
+	'is_wpforteams_site',
+	'updated_at',
+	'is_redirect',
+	'unmapped_url',
+] as const;
```

Comment:
It looks like these additional fields add nominal performance cost to the request:

Before:

```sh
$ php performance-tester.php 1629763 excerpts
Requesting /v1.1/me/sites?fields=ID,URL,is_coming_soon,is_private,launch_status,icon,name,options&options=is_wpforteams_site,updated_at
141 sites returned in 1.49s
```

After:

```sh
$ php performance-tester.php 1629763 excerpts
Requesting /v1.1/me/sites?fields=ID,URL,is_coming_soon,is_private,launch_status,icon,name,options,jetpack&options=is_wpforteams_site,updated_at,is_redirect,unmapped_url
141 sites returned in 1.491s
```

Updated the testing script with D84818-code

Reply:
I am a little worried that as we continue to build out the dashboard that we're going to need more and more data as part of the excerpt. I suppose we could split the excerpt into two requests e.g. we want to show the titles and `/home` links asap, but we can display the loading placeholder for `updated_at`.

But I feel like pagination would solve this issue properly ...

Reply:
> I suppose we could split the excerpt into two requests e.g. we want to show the titles and `/home` links asap, but we can display the loading placeholder for `updated_at`

Progressive enhancement would be _very nice_ 😄